analysis/skeleton_score_analysis.py

```diff
@@ -252,7 +253,10 @@ def get_default_result(_skeleton_idx: int) -> float:
 
 def _create_predicate_labels(
         predicate_set_order: Tuple[FrozenSet[Predicate], ...]) -> List[str]:
-    labels = [", ".join(p.name for p in predicate_set_order[0])]
+    if len(predicate_set_order[0]) > 5:
```

Comment:
is this a hack?

Reply:
yeah just for making the axis labels not crazy